src/server.tsx

```diff
@@ -48,27 +44,19 @@ app.get('/healthcheck', (req: express.Request, res: express.Response) => {
     res.send('OK');
 });
 
-interface Response {
-    html: string;
-    css: string;
-    js: string;
+interface Response2 {
```

Comment:
Is the '2' at the end a typo? (I don't have context for any other responses so just checking!)

Reply:
Ah, that was a renaming when I had both, that I should get rid of. (Though probably I will refactor this, depending on decision, to support new and old while we migrate.)